### PR TITLE
Set up STG

### PR DESCRIPTION
**Pull request by JacqueGM**
There has been various issues with the current setup for now this is what have been fixed:
- migrated from deprecated versions for ingress and cert-manager
- fixed issue with timeout with config from digital ocean https://github.com/digitalocean/terraform-provider-digitalocean/pull/704
- added two ingresses, one for each service as recommended in the setup https://kubernetes.github.io/ingress-nginx/user-guide/basic-usage/

17.2.2022
Fixing testing the following:
- [x] fix deployment for the API. It fails with an error  of authentication `FATAL:  no pg_hba.conf entry for host "10.10.10.5", user "doadmin", database "relayer", SSL off`  
- [x] re-test ingress installation 

Closes https://github.com/CirclesUBI/circles-docker/issues/25
Closes https://github.com/CirclesUBI/circles-iac/issues/10

Tasks:
- [x] Update version of [requirements](https://github.com/CirclesUBI/circles-iac#requirements) (e.g Terraform v1.1.7)
- [x] Fix relayer (update to latest)
- [x] Make consistent terraform local and tfstate in do.

### CODE REVIEW

**Review by llunaCreixent — COMMENTED** [review]
Looks good!

**Comment by JacqueGM** [comment]
Currently solving issues regarding DNS resolution and finalising TheGraph self-hosted setup

**Comment by llunaCreixent** [comment]
STG is up !! 🎊 
Now we are facing this issue: https://github.com/CirclesUBI/safe-relay-service/issues/52

**Comment by JacqueGM** [comment]
Removed TheGraph for now until we have a solid setup in place for it

**Comment by llunaCreixent** [comment]
There is a problem with sokol rpc node `wss://sokol.poa.network/wss`.
I'll be on top of it, but not available for now. I think we can merge this, and when the relayer is ready then update the service.

**Comment by llunaCreixent** [comment]
Is it a network problem that the relayer cannot connect to the rpc node?

**Comment by llunaCreixent** [comment]
I uploaded I new api image (`v1.3.22-007`), we should test it in staging env once the terraform issue is fixed.

**Comment by JacqueGM** [comment]
API version -> 1.4.0
relayer version --> 4.1.13

File: helm/circles-infra-suite/templates/configmap.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ data:
   GRAPH_NODE_ENDPOINT: {{.Values.graph.endpoint}}
   SUBGRAPH_NAME: {{.Values.graph.subgraph}}
 
-  # Ethereum Node Configuration
+  # Ethereum Node Configuration
```
Comment (llunaCreixent) [inline helm/circles-infra-suite/templates/configmap.yaml:26]: spaces

File: do-infra-setup/stg/main.tf [hunk 2]
```diff
@@ -127,9 +128,9 @@ resource "helm_release" "ingress" {
 
 resource "helm_release" "cert_manager" {
   name = "cert-manager"
-  repository = "https://charts.jetstack.io"
+  repository = "https://charts.jetstack.io"
```
Comment (llunaCreixent) [inline do-infra-setup/stg/main.tf]: spaces

**Review by llunaCreixent — COMMENTED** [review]
Finally working :)

**Review by llunaCreixent — APPROVED** [review]
🌷

**Pull request merged**